Sources/FileSystem/FileSystem.swift

```diff
@@ -314,9 +314,9 @@ public struct FileSystem: FileSysteming, Sendable {
 
         /// The path to the directory /var is a symlink to /var/private.
         /// NSTemporaryDirectory() returns the path to the symlink, so the logic here removes the symlink from it.
-        if systemTemporaryDirectory.starts(with: "/var/") {
-            systemTemporaryDirectory = "/private\(systemTemporaryDirectory)"
-        }
+//        if systemTemporaryDirectory.starts(with: "/var/") {
+//            systemTemporaryDirectory = "/private\(systemTemporaryDirectory)"
+//        }
```

Comment:
@fortmarek  Looks like you beat me to it but should we not just leave this as commented out code with no context. Either remove or do a OS check:

```swift
#if os(macOS)
if systemTemporaryDirectory.starts(with: "/var/") {
    systemTemporaryDirectory = "/private\(systemTemporaryDirectory)"
}
#endif
```